run_configure_ia.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```

Comment:
I think this file should be removed. It is a very specific build script that may confuse folks.

Reply:
I agree. I actually have a few of these, but accidentally this one made it to the repo.